src/Form/BasicDisable.php

```diff
@@ -11,7 +11,7 @@
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\Core\Url;
 use Drupal\tfa\TfaSetup;
-use Drupal\tfa_basic\Plugin\TfaValidation\TfaTotp;
+use Drupal\tfa\Plugin\TfaValidation\TfaTotp;
```

Comment:
I'd be curious on a case study for which we evaluate each of the files in TFA Basic and seeing what moves to TFA and what moves into a module like GA Login or TFA Basic